datadog/fwprovider/resource_datadog_integration_gcp_sts.go

```diff
@@ -309,6 +319,12 @@ func (r *integrationGcpStsResource) buildIntegrationGcpStsRequestBody(ctx contex
 	}
 	attributes.SetHostFilters(hostFilters)
 
+	runFilters := make([]string, 0)
+	if !state.CloudRunRevisionFilters.IsNull() {
+		diags.Append(state.CloudRunRevisionFilters.ElementsAs(ctx, &runFilters, false)...)
+	}
+	attributes.SetCloudRunRevisionFilters(runFilters)
```

Comment:
Same: sending an empty array.

Reply:
So if the CloudRunRevisionFilters is set, `ElementsAs(ctx, &runFilters, false)...)`

Will populate the TF Set with the values of the runFilters; in the event that `IsNull` is true then the intended behavior is to send an empty array

Reply:
https://github.com/DataDog/terraform-provider-datadog/pull/2612#discussion_r1797401223